libs/ledger-live-common/src/families/near/api/datahub-indexer-sdk.ts

```diff
@@ -0,0 +1,96 @@
+import { BigNumber } from "bignumber.js";
+import network from "../../../network";
+import { getEnv } from "../../../env";
+import { encodeOperationId } from "../../../operation";
+import { Operation, OperationType } from "@ledgerhq/types-live";
+import { NearTransaction } from "./sdk.types";
+
+const DEFAULT_TRANSACTIONS_LIMIT = 100;
+const getIndexerUrl = (route: string): string =>
+  `${getEnv("API_NEAR_INDEXER")}${route || ""}`;
+
+const fetchTransactions = async (
+  address: string,
+  limit: number = DEFAULT_TRANSACTIONS_LIMIT
+): Promise<NearTransaction[]> => {
+  const route = `/transactions?limit=${limit}&account=${address}`;
```

Comment:
I'm curious about not have a page parameter for an indexer ? So we can't see after 100 transactions right ?

Reply:
That's right, no page param, so we just fetch the latest 100 TXs.